project/dexter-monitor/routes/project.js

```diff
@@ -71,6 +75,68 @@ exports.saveSnapshotSummary = function() {
         })
 };
 
+exports.createProject = function(req, res) {
+
+	Promise.resolve(req.body.project)
+	.then(createDexterServerDatabase)
+	.then(insertDexterMonitorEntry)
+	.then((project) => {
+		startDexterServer(project);
+		
+		//wait for the dexter server to start
+		setTimeout( () => {
+			log.info('Dexter Server instance created at:' + project.hostName + ":" + project.portNumber );
+			res.send({status:"ok"});
+		}, 5000);
+    })
+	 .catch((err) => {
+        log.error(`Failed to create Dexter Server instance : ${err.message}`);
+        res.send({status:"fail", errorMessage: err.message});
+    });
+}
+
+function createDexterServerDatabase(project) {
+	var sql = `CREATE DATABASE ${project.projectName}; USE ${project.projectName};`;
```

Comment:
We must check the 'project.projectName' without '-', '#' and blank character.
The other way is to provide a space for the user to enter the database name.
▶AS_IS : dbName = ${project.projectName}
▶TO_BE : ${project.databaseName} or ${checkFunction(project.projectName)}